cyber-threat-intelligence/examples/05-infered-uses.tql

```diff
@@ -0,0 +1,36 @@
+# Example 05:
+
+# Now let's research the attack we described earlier using the id_unknown and threat_actor stix_id
+# This query represents the following question: "Who targets the unknown id and what tool is used?"
+
+match
+$t($x, $y) isa targets;
+$c($a, $b) isa attributed_to;
+$b has stix_id "bravo_threat";
+$x has stix_id "bravo_id_unknown";
+
+#--------------------------------------------------
+# Exercise 05:
+
+# When we run this query, we get a few results, but let's try toggling the inference button at the top to switch on
+# TypeDB's rule-inference. Now if we run the query again, we get many more results shown in a green outline. This is
+# because of a rule encoded into the schema:
+# 
+# rule transitive_use:
+# when {
+#     $x isa stix_domain_object, has name $name1;
```

Comment:
This rule needs to be updated to match the new version in the schema without names.